files/jsons/Policies/Policy_Table_Update/ptu_22734.json

```diff
@@ -0,0 +1,1274 @@
+                        "hmi_levels": [
+                            "BACKGROUND",
+                            "FULL",
+                            "LIMITED"
+                        ],
+                        "parameters": [
+                            "bodyInformation",
+                            "deviceStatus",
+                            "engineTorque",
+                            "externalTemperature",
+                            "fuelLevel",
+                            "fuelLevel_State",
+                            "headLampStatus",
+                            "instantFuelConsumption",
+                            "odometer",
+                            "tirePressure",
+                            "wiperStatus"
+                        ]
+                    },
+                    "UnsubscribeVehicleData": {
+                        "hmi_levels": [
+                            "BACKGROUND",
+                            "FULL",
+                            "LIMITED"
+                        ],
+                        "parameters": [
+                            "bodyInformation",
+                            "deviceStatus",
+                            "engineTorque",
+                            "externalTemperature",
+                            "fuelLevel",
+                            "fuelLevel_State",
+                            "headLampStatus",
+                            "instantFuelConsumption",
+                            "odometer",
+                            "tirePressure",
+                            "wiperStatus"
+                        ]
+                    }
+                },
+                "user_consent_prompt": "VehicleInfo"
+            }
+        },        
+        "module_config": {
+            "endpoints": {
+                "0x04": {
+                    "default": [
+                        "http://ivsu.software.ford.com/api/getsoftwareupdates"
+                    ]
+                },
+                "0x07": {
+                    "default": [
+                        "http://policies.telematics.ford.com/api/policies"
+                    ]
+                },
+                "lock_screen_icon_url": {
+                    "default": [
+                        "http://i.imgur.com/QwZ9uKG.png"
+                    ]
+                },
+                "queryAppsUrl": {
+                    "default": [
+                        "http://sdl.shaid.server"
+                    ]
+                }
+            },
+            "exchange_after_x_days": 30,
+            "exchange_after_x_ignition_cycles": 100,
+            "exchange_after_x_kilometers": 1800,
+            "notifications_per_minute_by_priority": {
+                "COMMUNICATION": 6,
+                "EMERGENCY": 60,
+                "NAVIGATION": 15,
+                "NONE": 0,
+                "NORMAL": 4,
+                "VOICECOM": 20
+            },
+            "preloaded_pt": false,
```

Comment:
@istoimenova, Please remove "preloaded_pt" since it has to be omitted in PTU file